socorro/processor/signature_utilities.py

```diff
@@ -564,19 +538,17 @@ def version(self):
         return '1.0'
 
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        return 'AbortMessage' in raw_crash and raw_crash.AbortMessage
+        return bool(raw_crash.get('AbortMessage'))
```

Comment:
curious. I rarely see `bool` used, even where we're clearly testing for a boolean. Should it be the convention going forward for `_predicate`s?